frontend/src/pages/edit-user-page/index.js

```diff
@@ -0,0 +1,29 @@
+import "../../index.css";
+import "./index.css";
+import React from "react";
+import NavBar from '../../components/navbar/index'
```

Comment:
better if we use absolute pathing instead. can you try if this works?
```suggestion
import NavBar from 'src/components/navbar/index'
```